plugins/login/src/components/MainLoginForm.svelte

```diff
@@ -0,0 +1,40 @@
+// Copyright © 2020 Anticrm Platform Contributors.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+<script lang="ts">
+  import login  from '..'
+  import { AnyComponent, ApplicationRoute, newRouter } from '@anticrm/platform-ui'
+  import Component from '@anticrm/platform-ui/src/components/Component.svelte'
+  
+  let form: ApplicationRoute
+  let forms: ApplicationRoute[] = [
```

Comment:
I think it could be a bit more simple.
```

import LoginForm from './LoginForm.svelte'
import SettingsForm from './LoginForm.svelte'
import { newRouter } from '@anticrm/platform-ui'

let isSettings = false
interface LoginOptions {
  extra: string
}
 newRouter<LoginOptions>(':route', (opt) => {     
    isSettings = "settings" === opt.extra 
 })
```

{#if isSettings}
<SettingsForm/>
{:else}
<LoginForm/>
{/if}

Reply:
I thought about it, but decided that it is necessary this way because it is more flexible and we will add more settings (for example, user information, logo, etc.)

Reply:
We should think about some platform tools for such sub-component routing. 
ApplicationRoute was created to work with metadata.